op-e2e/actions/helpers/l2_sequencer.go

```diff
@@ -104,7 +104,7 @@ func (s *L2Sequencer) ActL2StartBlock(t Testing) {
 		t.InvalidAction("already started building L2 block")
 		return
 	}
-	s.synchronousEvents.Emit(sequencing.SequencerActionEvent{})
+	s.synchronousEvents.Emit(sequencing.SequencerActionEvent{Ctx: event.WrapCtx(t.Ctx())})
```

Comment:
similarly, a helper `WithCtx(ctx)` could make this
```suggestion
	s.synchronousEvents.Emit(sequencing.SequencerActionEvent{}.WithCtx(t.Ctx()))
```